### PR TITLE
Adding "Type of attacks"+ "Existing solutions" sections

### PR DESCRIPTION
**Pull request by ambouhou**

### CODE REVIEW

File: State Ot Art.md [hunk 1]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
+* Ransom : Win32/CVE-2017-0147.A (121)
+* TrojanDownloader : Win32/ZombieBoy.A!bit (16)
+* Trojan : Win32/Occamy.CBO (12)
+* Trojan : Win32/Tiggre!r.fn (8)
+* Ransom : Win32/WannaCrypt.A!rsm (6)
+* Trojan : Win32/Eqtonex.F!rfn (2)
+
+They found that most of the malware encountered is ransoms or trojans/ TrojanDownloader. 
+
+## Measures of Effectiveness
```
Comment (banzo) [inline State Ot Art.md:215]: remove this?

File: State Ot Art.md [hunk 2]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
+The honeypot is placed in a specific IP address range and isolated from the company’s resources.
+All attempts to connect to the honeypot are considered malicious. The honeypot provides services such as SMB, HTTP, SSH, FTP, TFTP, MySQL, MSSQL, SIP, etc, depending on the characteristics defined above.
+Once the attackers' activities have been collected, this information must be processed in order to dissect the threats and possible ramifications of the attacks.
+This collected information can be attack sequences, commands, scripts and any other malicious payloads. They generate information on threats and thus enrich the organization’s risk analyses in order to mitigate those risks.
 
-In his article, Wenjun's Fan [^2] proposes a honey network taxonomy that facilitates their classification and study. 
-It is composed of 3 main features:
-Data control: Set of measures aimed at reducing the risk that a honeypot will be compromised and used to attack other systems. Outbound attacks must be controlled to protect non-honeypot systems
+### Threat Analysis
```
Comment (banzo) [inline State Ot Art.md:163]: define this term, using a standard (sourced) definition

File: State Ot Art.md [hunk 3]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
+This key attack information can be linked to a particular service. For example, the ssh service often starts with an authentication process and continues with sending a command or instruction that may involve using a URL to download a load used in order to exploit the host. In this case, the URL is an indicator of compromise and allows to highlight the importance of network services in the identification of the attacker, his intentions and his behavior.
 
-Data collection: Means necessary and used to guarantee the safe transfer of data collected by honeypots to a centralized point.
+Payload artifacts are parts of a program or malware used to perform various malicious activities in order to achieve the attacker’s objectives. These artifacts can be analyzed in 2 different ways. Or statically, by quickly analyzing the file structure of a program without executing it. The binary code is analyzed to determine if the signature is known and if it is malware. Be dynamically, running the code in a sandbox to discover the behavior of the program at execution.
```
Comment (banzo) [inline State Ot Art.md:169]: "or statically" 
seems it is missing sentence about dynamic analysis

File: State Ot Art.md [hunk 4]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
 
-## Type of attacks 
+All of these artifacts provide an easy-to-share knowledge base for cybersecurity organizations. One of the best known is Mitre.
 
 ### Mitre
```
Comment (banzo) [inline State Ot Art.md:173]: add links to mitre

File: State Ot Art.md [hunk 5]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
 
-Data collection: Means necessary and used to guarantee the safe transfer of data collected by honeypots to a centralized point.
+Payload artifacts are parts of a program or malware used to perform various malicious activities in order to achieve the attacker’s objectives. These artifacts can be analyzed in 2 different ways. Or statically, by quickly analyzing the file structure of a program without executing it. The binary code is analyzed to determine if the signature is known and if it is malware. Be dynamically, running the code in a sandbox to discover the behavior of the program at execution.
 
-## Type of attacks 
+All of these artifacts provide an easy-to-share knowledge base for cybersecurity organizations. One of the best known is Mitre.
 
 ### Mitre 
 MITRE is an American non-profit organization whose objective is to work for the public interest. His fields of intervention are systems engineering, information technology, operational concepts, and business modernization. Their slogan is that they solve problems for a safer world.
```
Comment (banzo) [inline State Ot Art.md:174]: his --> its
I'd remove the slogan

File: State Ot Art.md [hunk 6]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
 
 MITRE CVE [^9] is a public dictionary that records all vulnerabilities and security flaws that have been discovered. CVE identifiers are notably used by IDSs and IPSs.
 
 The use of these 2 sources of information by a honeypot can be interesting insofar as they provide interesting techniques to feed the attacker's killchain. 
-TODO->
 
-*Usage of Mitre (Honeypot dapted to certain threats) ? 
+### Categorization of threats
+As part of their research, Ryandy and Lim [^23] argues that data collected by honeypot can be processed and analyzed, using algorithms to correlate with Mitre’s threat scheme to finally categorize the various threats. They installed two well-known honeypots, namely Crowie and Dionaea. Over a two-month period in 2020, they were able to observe 547 million attacks. Of these attacks, only 1% were successful in logging in and executing commands or delivering a payload. Nevertheless, 96.1% were able to connect without executing any commands. They concluded that attackers spend most of their time scanning ports rather than trying to exploit the system.
```
Comment (banzo) [inline State Ot Art.md:185]: add links to Cowrie and Dionea (not as ref)

File: State Ot Art.md [hunk 7]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
 
 The use of these 2 sources of information by a honeypot can be interesting insofar as they provide interesting techniques to feed the attacker's killchain. 
-TODO->
 
-*Usage of Mitre (Honeypot dapted to certain threats) ? 
+### Categorization of threats
+As part of their research, Ryandy and Lim [^23] argues that data collected by honeypot can be processed and analyzed, using algorithms to correlate with Mitre’s threat scheme to finally categorize the various threats. They installed two well-known honeypots, namely Crowie and Dionaea. Over a two-month period in 2020, they were able to observe 547 million attacks. Of these attacks, only 1% were successful in logging in and executing commands or delivering a payload. Nevertheless, 96.1% were able to connect without executing any commands. They concluded that attackers spend most of their time scanning ports rather than trying to exploit the system. 
+
+It is interesting to note that they were able to observe a much higher number of attacks during working days than during the weekend. He suggests that this is a strong indicator that source IP addresses would come from desktop computers used as bots.
```
Comment (banzo) [inline State Ot Art.md:187]: this would heavily depend on the attacker timezone

File: State Ot Art.md [hunk 8]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
+<b> Fig.5 - Threat Categorization & Frequency Statistics </b> [^23]     
+
+In the case of Cowrie, more focused on artifacts, networks, the categorization of these threats could be done through the recovery of the various commands entered by the attackers: 
+![Basic Honeypot Exemple](/IMAGES/Command-Categorization.PNG)  
+<b> Fig.6 - Cowrie command & MITRE categorization </b> [^23]     
+
+When combined with the Mitre technique, these commands highlighted three categories of attacks, namely:
+* SCS005 (System Profiling & Persistence)
+* SCS029 (System Profiling)
+* SCS007 (Shell, Tool Exec. & System Profiling)
```
Comment (banzo) [inline State Ot Art.md:202]: add link to documentation on those categories

File: State Ot Art.md [hunk 9]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
+* Ransom : Win32/WannaCrypt.A!rsm (6)
+* Trojan : Win32/Eqtonex.F!rfn (2)
+
+They found that most of the malware encountered is ransoms or trojans/ TrojanDownloader. 
+
+## Measures of Effectiveness
+
+## Honeynet 
+The capabilities of a single honeypot are limited to its characteristics. Today, many honeypot systems focus on combining different types of honeypot to achieve an optimized solution. 
+A honeynet is an extension of the honeypot concept respecting a certain topology.
```
Comment (banzo) [inline State Ot Art.md:219]: this definition feels a bit abstract to me

File: State Ot Art.md [hunk 10]
```diff
@@ -152,43 +152,84 @@ In the case of a **production**, its purpose is above all to protect the real sy
...
-###  Lockheed-Martin Killchain
-*confront MITRE
+Data capture: Its purpose is to capture and record and collect the activities of attackers in order to be exploited later. 3 critical data capture layers have been identified:
+* The layer related to incoming and outgoing connections stored in the firewall logs.
+* The traffic analysis layer, which checks packets and their payloads as they enter and leave the honeynet.
+* The layer related to activity on the system (system calls, modified files, attacker's keystroke, etc).
+
+Data collection: Means necessary and used to guarantee the safe transfer of data collected by honeypots to a centralized point.
 
 ## Honeypot solutions referances
```
Comment (banzo) [inline State Ot Art.md:232]: referernces

File: State Ot Art.md [hunk 11]
```diff
@@ -217,12 +258,8 @@ https://github.com/telekom-security/tpotce
     - kako : https://github.com/darkarnium/kako  
     The default config will run a number of service simulations in order to capture attacking information from all incoming requests, including the full body. It includes Telnet, HTTP and HTTPS servers. Kako requires the following Python packages to work properly: Click, Boto3, Requests and Cerberus. Once you’re covered with the required packages, you can configure this IOT honeypot by using a simple YAML file called kako.yaml. All the data is recorded and is exported into AWS SNS, and flat-file JSON format.    
 
-
-
 # Honeypot and DevSecOps
```
Comment (banzo) [inline State Ot Art.md:261]: remove

Comment (ggi-cetic) [inline State Ot Art.md:261]: In DevSecOps file?

File: State Ot Art.md [hunk 12]
```diff
@@ -217,12 +258,8 @@ https://github.com/telekom-security/tpotce
...
     The default config will run a number of service simulations in order to capture attacking information from all incoming requests, including the full body. It includes Telnet, HTTP and HTTPS servers. Kako requires the following Python packages to work properly: Click, Boto3, Requests and Cerberus. Once you’re covered with the required packages, you can configure this IOT honeypot by using a simple YAML file called kako.yaml. All the data is recorded and is exported into AWS SNS, and flat-file JSON format.    
 
-
-
 # Honeypot and DevSecOps 
 
-## DevSecOps 
-
 ## Criteria for selecting a honeypot
 ### Definition of a strategy:
```
Comment (banzo) [inline State Ot Art.md:264]: do we keep this section (here)?

**Pull request merged**